style/sass/README.md

```diff
@@ -27,29 +27,178 @@
+
+<details>
+
+#### Code examples
+
+`h1.page-title` carries a specificity of 2, but can be reduced to 1 by removing
+the `h1` type selector:
+
+```diff
+-h1.page-title
++.page-title {
+   // …
+ }
+```
+
+#### Motivation
+
+Using an ID in a selector increases its specificity, making it more difficult
+to work with alongside class selectors. Furthermore, because IDs must be unique
+within an HTML document, using them as CSS selectors limits reusability.
+
+#### Resources
+
+- Learn about [how specificity is calculated][specificity-calculation].
+
+[specificity-calculation]: https://www.w3.org/TR/selectors-3/#specificity
+
+</details>
+
+### Selector naming
+
+- Use lowercase characters and hyphens (sometimes referred to as hyphen-case,
+  dash-case, or kebab-case) when naming selectors.
+- Be consistent about naming conventions. For instance, if a project is using
+  BEM, continue using it, and if it's not, don't introduce it.
+- Don't uses Sass parent selectors (`&`) to concatenate selector names.
+
+<details>
+
+#### Code examples
+
+Use lowercase characters and hyphens in selector names:
+
+```scss
+.class-name {
+  // …
+}
+```
+
+Don't concatenate selector names:
+
+```scss
+.class {
+  &__child-class {
+    // …
+  }
+}
+```
+
+#### Motivation
+
+Concatenating selector names makes it more difficult to search and find
+selectors in the codebase.
+
+</details>
+
+## General syntax and formatting
+
+### Declarations block ordering
+
+- Order declarations alphabetically.
+- Order items within the declaration block in the following order:
+    1. Sass at-rules, e.g. `@include`
+    1. CSS properties
+    1. Media queries
+    1. Pseudo-classes
+    1. Pseudo-elements
+    1. Nested elements
+
+<details>
+
+#### Code examples
+
+Alphabetize declarations:
+
+```scss
+.class {
+  display: block;
+  text-align: center;
+  width: 100%;
+}
+```
+
+Alphabetize prefixed properties as if the prefix doesn't exist:
+
+```scss
+.class {
+  font-family: system-ui;
+  -webkit-font-smoothing: antialiased;
+  font-weight: $weight-variable;
+}
+```
+
+Comprehensive example of ordering items within a declaration block:
+
+```scss
+.class {
+  @include size(10px);
+
+  display: block;
+  margin: $spacing-variable;
+
+  @media (min-width: $screen-variable) {
+    padding: $spacing-variable;
+  }
+
+  &:focus {
+    border-color: $color-variable;
+  }
+
+  &::before {
+    content: "";
+  }
+
+  .nested-element {
+    margin: $spacing-variable;
+  }
+}
+```
+
+#### Motivation
+
+Alphabetizing is automatable and is commonly a feature built into
+code editors (see Resources below).
+
+#### Linting
+
+Alphabetical declaration ordering can be linted using stylelint with the
+[stylelint-order][stylelint-order] plugin and its
+`order/properties-alphabetical-order` rule.
+
+[stylelint-order]: https://github.com/hudochenkov/stylelint-order
+
+#### Resources
+
+- Atom users can use the [Sort Lines package][sort-lines], which provides
```

Comment:
Again, awesome to include this stuff! It has made my life much easier and it's great that someone else could find these resources right here.